core/sql/exp/exp_clause_derived.h

```diff
@@ -1542,7 +1542,9 @@ enum conv_case_index {
   CONV_UTF8_F_UCS2_V                   =248,
 
   CONV_BLOB_BLOB                       =249,
-  CONV_BLOB_ASCII_F                    =250
+  CONV_BLOB_ASCII_F                    =250,
+
+  CONV_GBK_F_UTF8_V                    =251
```

Comment:
Do we also need a case for CONV_GBK_F_UTF8_F ? I am not quite how these case indexes work though. Maybe my question is if we have the user giving us GBK characters for a CHAR(N) UTF8 column, will it work with this change? Or is change applicable only when the target column in of type VARCHAR(N) UTF8